package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "express-ajv-swagger-validation",
-  "version": "1.2.0",
-  "description": "Input validation using Swagger (Open API) and ajv",
+  "name": "openapi-validator-middleware",
+  "version": "0.0.0",
```

Comment:
Why not start at 1.0.0?

Reply:
actually i'll preserve the old version as all the previous tags still exists.